services/alertmanager/registry_test.go

```diff
@@ -55,7 +55,7 @@ func TestRegistry(t *testing.T) {
 
 		expected := &ammodels.PostableAlert{
 			Annotations: annotations,
-			EndsAt:      strfmt.DateTime(nowValueCopy.Add(resolveTimeoutFactor * resendInterval)),
+			EndsAt:      strfmt.DateTime(nowValueCopy.Add(resolveTimeoutFactor * defaultResendInterval)),
```

Comment:
Seems like that right place to set `EndsAt`. We want alerts to live that time after we published them.